src/contacts.js

```diff
@@ -1,95 +1,21 @@
 import React, {Component, Fragment} from 'react';
-import PropTypes from 'prop-types';
 import classNames from "classnames";
-
 import {Language} from './App';
-import styles from "./ContactForm.css";
-
+import styles from "./style/ContactForm.css";
+import BigTextField from "./content/contactForm/BigTextField";
+import SmallTextField from "./content/contactForm/SmallTextField";
+import Button from "./content/contactForm/button";
 const loadText = "Loading";
-console.log(Language);
 
-function InputField(props) {
+export const InputField =(props) => {
```

Comment:
why basic InputField have you left in this file?

Reply:
because i use InputField in SmallTextField.js file